### DISCUSSION TITLE
Expand a section by mousse over & display dynamic text

### DISCUSSION THREAD

**Discussion by SadarSSI** [Lissy93/dashy#291] [Ideas]
Hello,
First of all, thank you for this wonderful dashy ;-))

Would it be possible to have these 2 possibilities in a future version : 
- expand a section by mouse over
- display dynamic text like @IP of the server or @IP of the client

I have not seen how in the doc to do this simply with the Dash configuration file. In a future version would it be possible to call a function/service ? (or equivalent, I'm not really a programmer) to display, scroll dynamically text

**Comment by lissy93** [comment]
Heya @SadarSSI
These are good ideas, thanks the suggestions :)

I'm planning a future feature which will allow for dynamic content, so I'll update you when that's implemented. Should be in a few weeks time. :) 

In terms of the hover to expand sections, I could make an option for this, but I think the UX for this might not be as good as it sounds in theory. There would probably need to be a sort delay, to prevent sections from opening and closing when your just moving the mouse around the screen, but if there's say a 0.5 sec delay, then clicking / tapping would be quicker anyway. Maybe I'll implement this in a PR and send you a link to a demo, and we can see if it works out okay.

**Reply by SadarSSI** [reply]
Yes, for the mouse-over I had not taken into account the sorting parameter which actually risks increasing the response times of the display. I was using dashmachine before and there was no this sorting feature.
We can consider that the fast display is unsorted and the slower display would take the sorting into account by letting the user choose their rendering / response time.
Personally, I consider that sorting my links in the section is not a constraint via editing the configuration file with a vscode editor or via the excellent interface offered ;-))
